modules/ocf/manifests/auth.pp

```diff
@@ -182,14 +182,25 @@
   }
 
   # sudo user/group access controls
-  package { 'sudo': }
+  package { ['sudo', 'libpam-google-authenticator', 'libpam-yubico']: }
```

Comment:
I think this might cause duplicate resource declaration problems since it's declared here and also in ocf_puppet. I'd just remove it from ocf_puppet since it's already here (and add a comment about it).